.gitignore

```diff
@@ -48,3 +48,4 @@ build-aux/compile
 build-aux/test-driver
 src/stamp-h1
 libsecp256k1.pc
+ecc-secp256k1
```

Comment:
nit: GitHub says "No newline at end of file"

Reply:
Didn't know newline at the end is the standard. fixed.